sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/statefulOperators.scala

```diff
@@ -324,6 +344,33 @@ object WatermarkSupport {
       }
     Some(evictionExpression)
   }
+
+  def findEventTimeColumn(
```

Comment:
Add method documentation since this is used outside.

Reply:
OK.